R/DiseasystoreBase.R

```diff
@@ -319,6 +319,13 @@ DiseasystoreBase <- R6::R6Class( # nolint: object_name_linter.
       # Determine the keys
       observable_keys  <- colnames(dplyr::select(observable_data, tidyselect::starts_with("key_")))
 
+      # Give warning if aggregation features are already in the observables data
+      existing_stratification <- intersect(names(observable_data), aggregation_features)
+      if (length(existing_stratification) > 0) {
```

Comment:
Instead of defining an object which will keep existing within the function scope, consider calling the `if` statement inline:

```
if (length(intersect(names(observable_data), aggregation_features)) > 0) {
#...
}
```